src/SonarScanner.MSBuild.Tasks/SonarScanner.MSBuild.Tasks.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Build" Version="15.9.20" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.9.20" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-    <PackageReference Include="System.Text.Json" Version="8.0.4" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
```

Comment:
Fixing the version to 8.0.5 should silence Mend. This is production code, but System.Text.Json is a transitive dependency, so there isn't much we can do about it. Also `[ExtensionData]` is not a very common API.
https://github.com/dotnet/runtime/issues/108678
I don't think we need to do a quick release with this change.